src/main/kotlin/community/flock/eco/workday/config/WirespecConfiguration.kt

```diff
@@ -1,12 +1,12 @@
 package community.flock.eco.workday.config
 
 import com.fasterxml.jackson.databind.Module
-import community.flock.wirespec.integration.jackson.WirespecModule
+import community.flock.wirespec.integration.jackson.kotlin.WirespecModuleKotlin
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 class WirespecConfiguration {
     @Bean
-    fun wirespecModule(): Module = WirespecModule()
+    fun wirespecModule(): Module = WirespecModuleKotlin()
```

Comment:
FYI: wirespec had a small breaking change in terms of package for the jackson integration